tests/ui/match_on_vec_items.rs

```diff
@@ -0,0 +1,130 @@
+#![warn(clippy::match_on_vec_items)]
+
+fn main() {
+    match_with_wildcard();
+    match_without_wildcard();
+    match_wildcard_and_action();
+    match_vec_ref();
+    match_with_get();
+    match_with_array();
+}
+
```

Comment:
Big NIT: Can you move this to the end of the file, so adding tests in the future doesn't require to update every line number in the reference file?